getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`, the {product-title} command line utility.
+
+If you have already installed the required prerequisites, here are the commands you need to create a cluster.
+
+[source, bash]
```

Comment:
While using `bash` is disallowed, I can suggest using the new `[source,terminal]` to see how that works.